.github/actions/nexus-associate-tag/action.yml

```diff
@@ -0,0 +1,42 @@
+name: Associate tag with artifacts
+description: Associate tag with artifacts in Nexus
+inputs:
+  tag-name:
+    description: The name of the tag
+    required: true
+  nexus-username:
+    description: Nexus user name
+    required: true
+  nexus-password:
+    description: Nexus password
+    required: true
+  nexus-url:
+    description: Base URL to the Nexus server
+    required: false
+    default: "https://artifacts.alfresco.com/nexus"
```

Comment:
better avoid hardcoding urls in input, we can add a global variable at the org level and use that in the downstream repositories